tasks/tarballs.yml

```diff
@@ -0,0 +1,133 @@
+---
+# vim: foldmarker=[[[,]]]:foldmethod=marker
+
+- name: Create Nextcloud group
+  group:
+    name: '{{ nextcloud__system_group }}'
+    state: 'present'
+    system: True
+  tags: [ 'role::nextcloud:download', 'role::nextcloud:verify' ]
+
+- name: Create Nextcloud user
+  user:
+    name: '{{ nextcloud__system_user }}'
+    group: '{{ nextcloud__system_group }}'
+    home: '{{ nextcloud__system_home }}'
+    comment: '{{ nextcloud__comment }}'
+    shell: '{{ nextcloud__shell }}'
+    system: True
+    state: 'present'
+  tags: [ 'role::nextcloud:download', 'role::nextcloud:verify' ]
+
+- name: Create source directory
+  file:
+    path: '{{ nextcloud__src }}'
+    state: 'directory'
+    owner: '{{ nextcloud__system_user }}'
+    group: '{{ nextcloud__system_group }}'
+    mode: '0755'
+
+- name: Create deployment directory
+  file:
+    path: '{{ owncloud__deploy_path }}'
+    state: 'directory'
+    owner: '{{ owncloud__app_user }}'
+    group: '{{ owncloud__app_group }}'
+    mode: '0750'
+
+
+# OpenPGP keys, application download and validation [[[1
+
+- become: True
+  become_user: '{{ nextcloud__system_user }}'
+  block:
+
+    # OpenPGP key management [[[2
+
+    - name: Ensure user GnuPG directory exists
+      file:
+        path: '{{ nextcloud__system_home + "/.gnupg" }}'
+        state: 'directory'
+        mode: '0700'
+
+    - name: Ensure the Nextcloud OpenPGP key is present
+      apt_key:
+        id: '{{ nextcloud__upstream_key_fingerprint | replace(" ","") }}'
+        keyring: '{{ nextcloud__system_home + "/.gnupg/pubring.gpg" }}'
+        keyserver: '{{ nextcloud__keyserver }}'
+        state: 'present'
+
+    # Get lastest patch version for release [[[2
+
+    - name: Query for the full version string of the current release
+      shell: curl -s -m 900 {{ (owncloud__variant_download_url_map[owncloud__variant] + "/") | quote }} | sed --silent 's/.*href="nextcloud-\({{ owncloud__release | regex_escape() }}[^"]\+\).zip.asc".*/\1/p' | sort --version-sort --reverse
+      args:
+        warn: False
+      register: nextcloud__register_full_version
```

Comment:
This variable is not used anywhere. Any plans for it? There are references for the '11.0.1' version later on, I guess this would have been replaced with the variable reference?